rdkit/Dbase/DbModule.py

```diff
@@ -11,17 +11,17 @@
 from rdkit import six
 from rdkit import RDConfig
 
-if hasattr(RDConfig, "usePgSQL") and RDConfig.usePgSQL:
-  from pyPgSQL import PgSQL
+if hasattr(RDConfig, "usePgSQL") and RDConfig.usePgSQL:  # pragma: nocover
+  from pyPgSQL import PgSQL  # @UnresolvedImport
```

Comment:
What the heck are all of these `@UnresolveImport` and `@UndefinedVariable` things?